### ISSUE TITLE
local server run fails to execute because the main website is down

### ISSUE THREAD

**Issue by mkarmona** [stanfordnlp/CoreNLP#1447]
@AngledLuffa The local version of CoreNLP isn't working because it's missing some JavaScript fetches from the main online website, which is currently down. What could be the potential solution?
- Provide a hosted version of those files?
- Or include them within the built package?

<img width="1906" alt="Screenshot 2024-05-22 at 13 36 00" src="https://github.com/stanfordnlp/CoreNLP/assets/5394232/a62ae41f-ddae-4e16-9e1f-205cf365ced2">

**Comment by mkarmona** [comment]
I see there a reference to same issue time ago [here](https://github.com/stanfordnlp/CoreNLP/issues/1356)

**Comment by AngledLuffa** [comment]
Yes, I can see the advantage of that (in particular, our own demo is
currently toast as well)

**Comment by AngledLuffa** [comment]
A big limitation here is that the CoreNLP server doesn't serve arbitrary pages.  I have to imagine that would be a big security risk.  Again I think the best solution is to wrap it in nginx or apache and serve the static files that way.  One limitation would be the landing page might need some edits to actually connect to the right host.  I can spend a little time working on that

**Comment by AngledLuffa** [comment]
I think I should be able to do this, actually, and just make the Brat JS libraries part of the regular CoreNLP download

https://github.com/stanfordnlp/CoreNLP/commit/5047ac1a2da44cac4cf4b8b1ac3d41ee32950abb

AJAX etc is a separate beast with lots of needed items.  I suppose someone who wants to be able to access the server on their own host without going to the outside world might be annoyed that it's not possible to cut that connection yet.  Still, I'm happy to make a new release with this improvement, at least
